src/api/spec/models/review_spec.rb

```diff
@@ -1,5 +1,161 @@
 require 'rails_helper'
 
+RSpec.shared_context 'some assigned reviews and some unassigned reviews' do
+  let!(:user) { create(:user) }
+
+  let!(:review_assigned1) { create(:review, by_user: user.login) }
+  let!(:review_assigned2) { create(:review, by_user: user.login) }
+  let!(:review_unassigned1) { create(:review, by_user: user.login) }
+  let!(:review_unassigned2) { create(:review, by_user: user.login) }
+
+  let!(:history_element1) do
+    create(:history_element_review_assigned, op_object_id: review_assigned1.id, user_id: user.id)
+  end
+  let!(:history_element2) do
+    create(:history_element_review_assigned, op_object_id: review_assigned2.id, user_id: user.id)
+  end
+  let!(:history_element3) do
+    create(:history_element_review_accepted, op_object_id: review_assigned2.id, user_id: user.id)
```

Comment:
Yes its supposed to be `review_unassined2.id` because `review_assigned2` needs to have multiple history elements in order to fully test the `.assigned` and `.unassigned` scopes. Before the `assigned` scope passed the test for a review with a single history_element of type `assigned` but failed for a review with multiple history elements, i.e. one history_elemnt_assigned and one history_element_accepted. Thats why I have it like this in the tests.